tools/ci/lua_stylecheck.py

```diff
@@ -92,9 +92,9 @@ def check_parameter_padding(self, line):
 
         See: TBD
         """
-        # ,[^ \n] : Any comma that does not have space or newline following
+        # ,[^ \n\"] : Any comma that does not have space or newline or close quote following
```

Comment:
@LoxleyXI Can you please explain the reason for this change? Also, please see the below note

Reply:
It allows the comma inside of a string `","`, which was necessary for `string.gsub` to remove them. Such a string could also be used to split. If prefer me to remove it, I can, but the PR will not pass CI checks without it.

Reply:
If you rebase to latest, it will pass without needing to change the checks

Reply:
I've just seen the change. Understandably, this didn't exist when I added this. Thanks.

Reply:
Looks like the comment line still needs to be changed back to reflect its original info

Reply:
Sorry, good catch. Updated.